db/fixtures/miq_product_features.yml

```diff
@@ -4612,125 +4612,117 @@
     :children:
-    - :name: View
-      :description: View Providers, Configuration Profiles, Configured Systems
+    - :name: List
+      :description: Display Lists of Configuration Manager Providers
       :feature_type: view
-      :identifier: provider_foreman_view
-    - :name: Operate
-      :description: Perform Operations on Providers and Configured Systems
+      :identifier: ems_configuration_show_list
+    - :name: Show
+      :description: Display Individual Configuration Manager Provider
+      :feature_type: view
+      :identifier: ems_configuration_show
+  - :name: Operate
+    :description: Perform Operations on Configuration Manager Providers and Configured Systems
+    :feature_type: control
+    :identifier: ems_configuration_control
+    :children:
+    - :name: Edit Tags
+      :description: Edit Tags of Configuration Manager Provider
       :feature_type: control
-      :identifier: provider_foreman_control
-      :children:
-      - :name: Edit Tags
-        :description: Edit Tags of Foreman Provider
-        :feature_type: control
-        :identifier: configuration_manager_provider_tag
-      - :name: Provision Configured Systems
-        :description: Provision Configured Systems
-        :feature_type: control
-        :identifier: provider_foreman_configured_system_provision
-      - :name: Refresh Providers
-        :description: Refresh relationships for all items related of Provider
-        :feature_type: control
-        :identifier: provider_foreman_refresh_provider
-    - :name: Modify
-      :description: Modify Providers
+      :identifier: ems_configuration_tag
+    - :name: Provision Configured Systems
+      :description: Provision Configured Systems
+      :feature_type: control
+      :identifier: ems_configuration_configured_system_provision
+    - :name: Refresh Configuration Manager Providers
+      :description: Refresh relationships for all items related to Configuration Manager Provider
+      :feature_type: control
+      :identifier: ems_configuration_refresh_provider
+  - :name: Modify
+    :description: Modify Configuration Manager Providers
+    :feature_type: admin
+    :identifier: ems_configuration_admin
+    :children:
+    - :name: Remove Configuration Manager Providers
+      :description: Remove Configuration Manager Provider
       :feature_type: admin
-      :identifier: provider_admin
-      :children:
-      - :name: Remove Providers
-        :description: Remove Provier
-        :feature_type: admin
-        :identifier: provider_foreman_delete_provider
-      - :name: Edit Provider
-        :description: Edit a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_edit_provider
-      - :name: Pause
-        :description: Pause a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_pause
-      - :name: Resume
-        :description: Resume a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_resume
```

Comment:
These feature were [removed](https://github.com/ManageIQ/manageiq-ui-classic/pull/6067) from UI but were not removed from product features

Reply:
@skateman about renaming the features to suggested names in the table above, i recall having some issues when i tried to rename those but the thing is that those are using common methods with automation manager and the features there are named similarly, it will require some code changes in that area too if i rename these. So I am planning to keep these unchanged.

Reply:
Okay, I'll work on the migration based on the table + your reaction.